### DISCUSSION TITLE
npm ERR! 405 Not Allowed - PUT

### DISCUSSION THREAD

**Discussion by towry** [verdaccio/verdaccio#3768] [Q&A]
I was publishing a package to the verdaccio server, got this error:

```
 error npm ERR! 405 Not Allowed - PUT https://verdaccio.x.com/@x%2feslint-config-x
🦋  error {
🦋  error   "error": {
🦋  error     "code": "E405",
🦋  error     "summary": "405 Not Allowed - PUT https://verdaccio.x.com/@x%2feslint-config-x",
🦋  error     "detail": ""
🦋  error   }
🦋  error }
```

What could be wrong ??

**Comment by juanpicado** [comment]
Welll ... you need to expand your info otherwise hard to know

**Reply by towry** [reply]
Hi, thanks for the quick reply.

I just found out that if the package( for example,`@x/abc`) contains dependencies like `@vue/cli`, then it will get this error. If I removed all scoped packages in the `@x/abc`, then it will be published success.

**Reply by towry** [reply]
Just found out, if the package contains `@type/abc`, or `@typesssssss/abc`, it will get this 405 error. only on remote server with https domain.

**Reply by towry** [reply]
https://stackoverflow.com/questions/76116131/verdaccio-put-405-not-allowed-error-when-publish-scoped-package?noredirect=1#comment134236612_76116131

**Reply by ChrisMiaoMiao** [reply]
I get the same error. if the package contains `@typexxxx/xxxxxx' , i get the 405 when publish. how can i fix it?
